test/new_relic/agent/transaction/request_attributes_test.rb

```diff
@@ -77,6 +147,111 @@ def test_sets_method_from_request
 
           assert_equal 'POST', attrs.request_method
         end
+
+        def test_by_default_only_a_base_set_of_request_headers_are_captured
+          skip_unless_minitest5_or_above
+
+          with_config(allow_all_headers: false, :'attributes.include' => [], :'attributes.exclude' => []) do
+            attrs = RequestAttributes.new(RACK_REQUEST)
+
+            BASE_HEADERS_MAP.each do |method, definition|
+              assert_equal definition[:value],
+                attrs.send(method),
+                "Expected RequestAttributes##{method} to yield >#{definition[:value]}<, got >#{attrs.send(method)}<"
+            end
+            assert_equal NewRelic::EMPTY_HASH, attrs.other_headers, 'Did not expect to find other headers'
```

Comment:
Kinda surprised rubocop didn't yell at not having a blank line before `assert`

Reply:
RuboCop is cool with multiple assertions grouped together, as long as there's a blank line between the group and the previous non-assertion code.

```ruby
result = do_something

assert result.is_cool?
assert result.size > MIN_SIZE
```

and here even though the actual words "assert" are more than 1 line apart, it's really just 2 assertions grouped together.